Assets/scss/components/_newfooter.scss

```diff
@@ -0,0 +1,177 @@
+.cmp-new-footer{
+    background-color: $color-grey-silver;
+    padding: 0 24px;
+
+    &__main-container{
+        max-width: toRem(1440);
+        margin: 0 auto;
+        padding: toRem(48) 0;
+    }
+
+    &__top-section{
+        display: flex;
+        flex-direction: column;
+        &-items{
+            padding: toRem(36) 0 0;
+            @include min-screen(xl){
+                width: 25%;
+                padding: 0 toRem(24);
+            }
+        }
+
+        @include min-screen(xl){
+            flex-direction: row;
+            width: calc(100% + toRem(48));
+            margin: 0 -24px;
+        }
+    }
+    &__title{
+        font-size: toRem(16);
+        color: $color-nero;
+        padding-block-end: toRem(12);
+        text-transform: uppercase;
+    }
+    &__link-section{
+        padding: toRem(16) 0 !important;
```

Comment:
Generally discourage the use of `!important` and as long as there aren't other instances of `.cmp-new-footer__link-section` (I see one on line 143, but there's no padding) then you shouldn't need to use this.

Reply:
There shouldn't be anything here that requires an !important directive. You probably will need to look at specificity to fix whatever is causing you to use it.